client/my-sites/stats/activity-log-item/activity-title.jsx

```diff
@@ -0,0 +1,289 @@
+import { get } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+class ActivityTitle extends Component {
+
+	static propTypes = {
+		action: PropTypes.string,
+
+		actor: PropTypes.shape( {
+			display_name: PropTypes.string,
+			login: PropTypes.string,
+		} ),
+
+		group: PropTypes.oneOf( [
+			'attachment',
+			'comment',
+			'core',
+			'menu',
+			'plugin',
+			'post',
+			'term',
+			'theme',
+			'user',
+			'widget',
+		] ).isRequired,
+		name: PropTypes.string.isRequired,
+
+		object: PropTypes.shape( {
+			attachment: PropTypes.shape( {
+				mime_type: PropTypes.string,
+				id: PropTypes.number.isRequired,
+				title: PropTypes.string.isRequired,
+				url: PropTypes.shape( {
+					host: PropTypes.string.isRequired,
+					url: PropTypes.string.isRequired,
+					host_reversed: PropTypes.string.isRequired,
+				} ).isRequired,
+			} ),
+
+			comment: PropTypes.shape( {
+				approved: PropTypes.bool.isRequired,
+				id: PropTypes.number.isRequired,
+			} ),
+
+			core: PropTypes.shape( {
+				new_version: PropTypes.string,
+				old_version: PropTypes.string,
+			} ),
+
+			plugin: PropTypes.oneOfType( [
+				PropTypes.shape( {
+					name: PropTypes.string,
+					previous_version: PropTypes.string,
+					slug: PropTypes.string,
+					version: PropTypes.string,
+				} ),
+				PropTypes.arrayOf(
+					PropTypes.shape( {
+						name: PropTypes.string,
+						previous_version: PropTypes.string,
+						slug: PropTypes.string,
+						version: PropTypes.string,
+					} ),
+				),
+			] ),
+
+			post: PropTypes.shape( {
+				id: PropTypes.number.isRequired,
+				status: PropTypes.string.isRequired,
+				type: PropTypes.string,
+				title: PropTypes.string,
+			} ),
+
+			term: PropTypes.shape( {
+				id: PropTypes.number.isRequired,
+				title: PropTypes.string.isRequired,
+				type: PropTypes.string.isRequired,
+			} ),
+
+			theme: PropTypes.oneOfType( [
+				PropTypes.arrayOf(
+					PropTypes.shape( {
+						name: PropTypes.string,
+						slug: PropTypes.string,
+						uri: PropTypes.string,
+						version: PropTypes.string,
+					} )
+				),
+				PropTypes.shape( {
+					name: PropTypes.string,
+					slug: PropTypes.string,
+					uri: PropTypes.string,
+					version: PropTypes.string,
+				} ),
+			] ),
+
+			user: PropTypes.shape( {
+				display_name: PropTypes.string,
+				external_user_id: PropTypes.string,
+				login: PropTypes.string,
+				wpcom_user_id: PropTypes.number,
+			} ),
+
+			widget: PropTypes.shape( {
+				id: PropTypes.number,
+				name: PropTypes.string,
+				sidebar: PropTypes.string,
+			} ),
+		} ),
+
+		// localize
+		moment: PropTypes.func.isRequired,
+		translate: PropTypes.func.isRequired,
+	};
+
+	getAction() {
+		return this.props.action || this.props.name;
+	}
+
+	getActorName() {
+		const { actor } = this.props;
+		const displayName = get( actor, 'display_name' );
+		if ( displayName ) {
+			return displayName;
+		}
+		const login = get( actor, 'display_name' );
```

Comment:
Whoops, thanks for catching that. Fixed in 9a27eb5e760bd8c019c741a31defaf306231cc8f